Spec/RealtimeClientConnection.swift

```diff
@@ -2057,13 +2060,16 @@ class RealtimeClientConnection: QuickSpec {
                             }
                         }
                         expect(channel.state).to(equal(ARTRealtimeChannelState.Detached))
-                        expect(channel.errorReason!.code).to(equal(80008))
-                        expect(channel.errorReason!.message).to(contain("Unable to recover connection"))
+                        guard let channelError = channel.errorReason else {
+                            fail("Channel error is nil"); return
```

Comment:
Btw. Don't change this, but in future we should have an error message that explains the problem rather than the boolean logic i.e. anyone can see from this code that channel error is nil, but what the reader cares about is why `nil` is a problem i.e. this should instead be `fail("Connection resume failed and error should be propagated to the cannel")`.   This helps the reader of the tests in future understand why this is a problem as opposed to just seeing that it is expected to be `nil`.